### PR TITLE
fixed ansi scape in regex

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

**Comment by rvagg** [comment]
oh! very interesting, so it sounds like git isn't picking up that it's not a tty, I have some style nits before I can merge this, will comment inline

File: commit-stream.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-const through2 = require('through2')
+const through2 = require('through2'),
+stripAnsi = require('strip-ansi');
```
Comment (rvagg) [inline commit-stream.js]: comma first, no semicolon, indent `stripAnsi` to line up at the start of `through2`, align `=`

```js
const through2  = require('through2'),
    , stripAnsi = require('strip-ansi')
```

File: commit-stream.js [hunk 2]
```diff
@@ -10,6 +11,8 @@ function commitStream (ghUser, ghProject) {
   return through2.obj(onLine, onEnd)
 
   function addLine (line) {
+    line = stripAnsi(line);
```
Comment (rvagg) [inline commit-stream.js]: remove semicolon

File: package.json [hunk 3]
```diff
@@ -17,11 +15,14 @@
     "list-stream": "~1.0.0",
     "minimist": "~1.1.0",
     "split2": "~0.2.1",
+    "strip-ansi": "^2.0.1",
```
Comment (rvagg) [inline package.json]: No `^` please, it needs to die in a fire. Also the changes introduced here are huge, perhaps something npm is doing now? very odd, but can you restore it to original and manually add in the `strip-ansi` line to make the impact less dramatic please?

**Comment by Kikobeats** [comment]
fixed!

sorry for the changes in `package.json`. I normally use a tool to normalize the keys inside the file. It's a little offtopic, but can you say me your opinion about use [finepack library](https://github.com/Kikobeats/finepack)? ([We can update how to sort the keys](https://github.com/Kikobeats/finepack/blob/master/lib/Keywords.coffee#L16))

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "changelog-maker",
-  "version": "1.2.4",
+  "version": "1.2.5",
```
Comment (rvagg) [inline package.json]: one last nit: please don't version bump, I use `npm version` to do this and it signs releases git tags for me in the process.

**Comment by rvagg** [comment]
would you mind squashing these commits into one given the amount of noise in there? sorry for the hassle.

**Pull request closed without merging**